contents/tutorials/react-cookie-banner.md

```diff
@@ -65,37 +67,78 @@ ReactDOM.createRoot(document.getElementById('root')).render(
     </PostHogProvider>
   </React.StrictMode>,
 )
-
 ```
 
 Once done, visit your site running locally, click the button, and you should start seeing events in your [PostHog activity tab](https://us.posthog.com/activity/explore).
 
 <ProductScreenshot
-    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_07_10_at_14_54_53_2x_de2d28eea9.png" 
-    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_07_10_at_14_54_41_2x_13e7233065.png"
-    alt="Events in PostHog" 
-    classes="rounded"
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_07_10_at_14_54_53_2x_de2d28eea9.png" 
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_07_10_at_14_54_41_2x_13e7233065.png"
+  alt="Events in PostHog" 
+  classes="rounded"
 />
 
 When we head back to our local site, right-click, choose inspect, go to the Application tab, and check cookies, you’ll see we’ve created a tracking cookie with details about PostHog and the user. 
 
 ![App cookie](https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2024_07_10_at_14_59_04_2x_19bbe1f339.png)
 
-## Step 3: Building the banner component
+## Step 3: Ensuring cookies aren't set on initial load
+
+To start making this more compliant, we want to ensure we don't store any cookies until the user opts in. 
```

Comment:
Nit, but maybe we just say "making this fully compliant" instead of "more compliant"

I think more/less compliant isn't the right wording here. I _think_ you are either compliant or you aren't. Ignore me if you have ideas otherwise :)